sdk/nodejs/provisioning/bin.ts

```diff
@@ -175,7 +175,8 @@ export class Bin implements EngineConn {
       )
     }
     throw new EngineSessionEOFError(
-      "No line was found to parse the engine connect params"
+      "No line was found to parse the engine connect params." +
```

Comment:
LGTM on Node SDK :rocket:

Reply:
nit: add an extra space after `params.` otherwise it will look like this : `No line was found to parse the engine connect params.If engine was manually stopped, run it again or delete it`. You can also add an extra `\n` if you want the text in two lines. (`"No line was found to parse the engine connect params. \n" +`)